vivisect/symboliks/common.py

```diff
@@ -81,6 +81,7 @@ def __init__(self):
         self.kids          = []
         self.parents       = []
         self.cache         = {}
+        self._reduced       = False
```

Comment:
probably not, the existing behavior would have to be preserved.  requiring some other user to become spontaneously aware of the need to call that function if they are adding nodes after reduce is likely to create heisenbugs.  the more i think about it, the more i like the walkTree(...,once=True) option. it'd be reasonably simple ( backward compatible ) and could just keep a visited dict and ditch it on return.